enterprise_access/apps/api_client/tests/test_license_manager_client.py

```diff
@@ -38,3 +45,208 @@ def test_get_subscription(self, mock_oauth_client):
             expected_url,
             timeout=settings.LICENSE_MANAGER_CLIENT_TIMEOUT,
         )
+
+
+class TestLicenseManagerUserApiClient(TestCase):
+    """
+     Test License Manager with BaseUserApiClient.
+     """
+    def setUp(self):
+        super().setUp()
+        self.api_base_url = 'http://license-manager.example.com/api/v1/'
```

Comment:
nit: I believe `LicenseManagerUserApiClient.api_base_url` might be possible to use here?